dash/CMakeLists.txt

```diff
@@ -212,6 +212,15 @@ if (ENABLE_SCALAPACK)
   endif()
 endif()
 
+if (HAVE_STD_TRIVIALLY_COPYABLE)
+  set (ADDITIONAL_COMPILE_FLAGS
+       ${ADDITIONAL_COMPILE_FLAGS} -DSTD_TRIVIALLY_COPYABLE_AVAIL)
```

Comment:
I suggest to use the `DASH_` prefix for macros, i.e., `DASH_HAVE_STD_TRIVIALLY_COPYABLE`, and similar below.